CHANGELOG.md

```diff
@@ -4,15 +4,16 @@ All notable changes to rpgp will be documented in this file.
 
 ## [0.12.0-alpha.2](https://github.com/rpgp/rpgp/compare/v0.12.0-alpha.1..0.12.0-alpha.2) - 2024-04-07
 
-### ⛰️  Features
```

Comment:
the changelog is auto regenerated on release, so not really much point of fixing anything

Reply:
right, will exclude